src/riak_core_coverage_fsm.erl

```diff
@@ -87,28 +85,30 @@
          terminate/3,
          code_change/4]).
 
--spec behaviour_info(atom()) -> 'undefined' | [{atom(), arity()}].
-behaviour_info(callbacks) ->
-    [
-     {init, 2},
-     {process_results, 2},
-     {finish, 2}
-    ];
-behaviour_info(_) ->
-    undefined.
+-type mod_state() :: term().
```

Comment:
It's curious that the type of `mod_state` in the state record is `tuple()`. Should this match `mod_state()`?